src/commands/appSettings/downloadAppSettings.ts

```diff
@@ -13,67 +13,52 @@ import { ext } from "../../extensionVariables";
 import { getLocalSettingsJson, ILocalSettingsJson } from "../../funcConfig/local.settings";
 import { localize } from "../../localize";
 import * as api from '../../vscode-azurefunctions.api';
-import { confirmOverwriteSettings } from "./confirmOverwriteSettings";
```

Comment:
Is this a file we own? if so, is it still needed? as in the actual confirmOverwriteSettings implementation.

Reply:
This was the old file that was used for the overwriting of settings during the download and upload flows, but is now no longer called upon so I can just go ahead and delete it.